src/main/java/com/capitalone/dashboard/model/CollectorType.java

```diff
@@ -20,6 +20,7 @@
     Scope,
     CodeQuality,
     Test,
+    TestSync,
```

Comment:
Can you explain the need for this new Collector Type?

Reply:
@rvema TestSync is removed, using Test as discussed.